### DISCUSSION TITLE
[request] More Bungeecord support in CMI

### DISCUSSION THREAD

**Discussion by wanhe886** [Zrips/CMI#578] [Feature Requests]
Since CMI already uses SQL, I think it will be really great for CMI to take full advantage from SQL storage.

If server owner enables SQL in their bungee cord servers, CMI should act like a pan-bungeecord plugin. Like banning player from server 1 means banning across the board. This also applies to muting, kits cool down, playtime rewards, and what not.
Currently one player has multiple lines in CMI's SQL, if multiple server enables SQL for CMI, it is a waste of storage space and waste of opportunity.

There is also a bug relates to using SQL storage, that the homes in different server are over writing each other in the CMI SQL, if CMI is more bungee cord compatible, I am sure this bug will also get fix.

**Comment by tankinnwow** [comment]
I would like to see move bungee support as well. I have five server behind my proxy and i would like to have global ranks, economy, tab list, and playtime.

I just started using CMI and i love most of it's features. Having greater network support would be a nice addition.

**Comment by ghost** [comment]
Support for cross-server teleportation would be nice also.
- /tp, /tpa, /tppos etc.
- Warps & Homes
- Respawn
